src/System.IO.FileSystem/src/System/IO/EnumerationOptions.cs

```diff
@@ -17,11 +17,21 @@ public class EnumerationOptions
         /// For internal use. These are the options we want to use if calling the existing Directory/File APIs where you don't
         /// explicitly specify EnumerationOptions.
         /// </summary>
-        internal static EnumerationOptions Compatible { get; } = new EnumerationOptions
-            { MatchType = MatchType.Win32, AttributesToSkip = 0, IgnoreInaccessible = false };
+        internal static EnumerationOptions Compatible
+        {
+            get
+            {
+                return new EnumerationOptions { MatchType = MatchType.Win32, AttributesToSkip = 0, IgnoreInaccessible = false, IsNormalized = false };
+            }
+        }
```

Comment:
We're now allocating a new object every time these are accessed.  That's not a problem?

Reply:
I'm not even sure we should change these as the default is what we want anyway. We don't explicitly set all of the members.

Reply:
Also, I'm guessing @carlossanlop you changed these because you modified the enumeration code to mutate the instances?  That seems wrong to me as well.  As a consumer of the API, I wouldn't expect that my options object would get mutated.  It's also not clear to me how that works when the same options object is used on multiple calls.

Reply:
This is mostly my fault. When I looked at where we could pass state I didn't look close enough and forgot that I had these cached instances. 😊

Reply:
Thanks for your comment @stephentoub, I now realize the mistake. Jeremy and I talked about this and we are going to follow a different approach that doesn't depend on modifying the cached instances. I will submit an iteration soon.

Reply:
Sounds good. Thanks.

Reply:
@JeremyKuhne @stephentoub done! Please take a look whenever you get a chance. I described my change in the commit message. All unit tests passed locally, I'm now waiting for the CI job to confirm the same.

Reply:
I have a combination of CI failures:
- Most of them were caused by existing Networking issues, I belive @krwq is fixing it or got it fixed already.
- The build failed in some platforms. I suspect I need to modify some platform specific files that did not show in VS as errors.